src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -279,18 +279,26 @@ def extra_resources(self, compile_context):
 
     return result
 
-  def extra_resources_digest(self, compile_context):
-    """Compute a Digest for the extra_resources for the given context."""
+  def extra_post_compile_resources_digest(self, compile_context):
```

Comment:
Very nit: The `_extra_` in the name is in a different position from in the other method.

Reply:
fixed.